sdk/devcenter/Azure.Developer.DevCenter/tests/DevBoxClientTests.cs

```diff
@@ -0,0 +1,415 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Developer.DevCenter.Tests
+{
+    [PlaybackOnly("It takes roughly 15-30 minutes to create a dev-box")]
+    public class DevBoxClientTests : RecordedTestBase<DevCenterClientTestEnvironment>
+    {
+        public const string DevBoxName = "MyDevBox";
+
+        private DevBoxesClient _devBoxesClient;
+
+        internal DevBoxesClient GetDevBoxesClient() =>
+            InstrumentClient(new DevBoxesClient(
+                TestEnvironment.Endpoint,
+                TestEnvironment.Credential,
+                InstrumentClientOptions(new DevCenterClientOptions())));
+
+        public DevBoxClientTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _devBoxesClient = GetDevBoxesClient();
+        }
+
+        [OneTimeTearDown]
+        public async Task OneTimeTearDown()
+        {
+            Operation devBoxDeleteOperation = await _devBoxesClient.DeleteDevBoxAsync(
+                WaitUntil.Completed,
+                TestEnvironment.ProjectName,
+                DevBoxName,
+                userId: TestEnvironment.UserId);
+
+            await devBoxDeleteOperation.WaitForCompletionResponseAsync();
+            Console.WriteLine($"Completed dev box deletion.");
+        }
+
+        [RecordedTest]
+        public async Task StartDevBoxSucceeds()
+        {
+            await SetUpDevBoxAsync();
+
+            // Start the dev box
+            Operation<BinaryData> devBoxStartOperation = await _devBoxesClient.StartDevBoxAsync(
```

Comment:
I believe this test is performing start on a dev box that's already in a running state, yeah? It may be a more interesting/meaningful validation to perform start on a stopped dev box.

Reply:
Fixed: https://github.com/Azure/azure-sdk-for-net/pull/36245/commits/dc89ddadb313688e8d91d87c78693898a53c456b